folium/plugins/mouse_position.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+import json
+
+from branca.element import CssLink, Figure, JavascriptLink, MacroElement
+
+from jinja2 import Template
+
+
+class MousePosition(MacroElement):
+    """
+    Adds a mouse position widget on the map.
+
+    Parameters
+    ----------
+    position: location of the widget
+        default is 'bottomright'.
+
+    CONVERTED TO PYTHON FROM:
+
+    https://github.com/ardhi/Leaflet.MousePosition
+
+    With the MIT License as below:
```

Comment:
Good that you include the license. But could you put in in the file header? The docstring is converted to documentation and I don't want that littered with license texts.